src/test/java/org/broadinstitute/hellbender/tools/examples/ExampleVariantWalkerIntegrationTest.java

```diff
@@ -0,0 +1,32 @@
+package org.broadinstitute.hellbender.tools.examples;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.tools.IntegrationTestSpec;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+public final class ExampleVariantWalkerIntegrationTest extends CommandLineProgramTest {
+    @Override
+    public String getTestedClassName() {
+        return ExampleVariantWalker.class.getSimpleName();
+    }
```

Comment:
It's an example of how to write a test class though -- we definitely should not have unnecessary and confusing stuff like this here!